src/components/map/Map.tsx

```diff
@@ -19,24 +20,32 @@ type Props = {
   onMaximizeButtonPress?: () => void;
   showsUserLocation?: boolean;
   style?: StyleProp<ViewStyle>;
-  zoom?: number;
 };
 
 const MARKER_ICON_SIZE = normalize(40);
 
 export const Map = ({
   geometryTourData,
   isMaximizeButtonVisible,
+  isMultipleMarkersMap = false,
   locations,
   mapCenterPosition,
   mapStyle,
   onMapPress,
   onMarkerPress,
   onMaximizeButtonPress,
   style,
-  zoom = 0,
   ...otherProps
 }: Props) => {
+  const { globalSettings } = useContext(SettingsContext);
+  const { settings = {} } = globalSettings;
+  const { zoomLevelForMaps = {}, locationService = {} } = settings;
+
+  const showsUserLocation = otherProps.showsUserLocation ?? !!locationService;
+  const zoom = isMultipleMarkersMap
+    ? zoomLevelForMaps?.multipleMarkers
+    : zoomLevelForMaps?.singleMarker;
```

Comment:
if you fallback to `{}` i guess you do not have to use the conditional chaining here, because `zoomLevelForMaps` is always present an accessible as object.

Reply:
Yeah, that's right. I fixed it.

https://github.com/smart-village-solutions/smart-village-app-app/commit/9710bf33bdc67175807eab8b3c663c27fcca6fc5